FogCreek/Glitch-Community
# weak-sale: componentize user options pop and create team pop

## Links
* [weak-sale.glitch.me](https://weak-sale.glitch.me)
* [Manuscript](https://glitch.fogbugz.com/f/cases/3328478/Build-User-Options-Pop)

## GIF/Screenshots:
<div>
<img src="https://user-images.githubusercontent.com/7584833/59626693-85ae3d80-910a-11e9-908b-5c667b9e8dcd.png" width="150" style="margin-right: 10px" />
<img src="https://user-images.githubusercontent.com/7584833/59626694-85ae3d80-910a-11e9-9444-ce606b8392d5.png" width="150" />
</div>

## Changes:
- Move user options pop and create team pop to `/components`
- Get rid of `:global` styles in the header that were styling the user avatar
- Use Hooks for create team pop
- Get rid of a few unused popover presenters

## How To Test:
* Open the user options pop, is it keyboard-navigable? Do all the buttons and links work?
* Open the create team pop. Type a team name that's already taken, you should see an error. Create a team, it should work.
